services/apps/alcs/src/providers/typeorm/migrations/1707848440329-set_purpose_on_app_noi_from_oats.ts

```diff
@@ -0,0 +1,46 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SetPurposeOnAppNoiFromOats1707848440329
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            WITH alcs_app_sub_to_update AS (
+            SELECT
+                as2.file_number
+            FROM
+                alcs.application_submission as2
+            WHERE
+                audit_created_by = 'oats_etl'
+                AND purpose IS NULL
+                AND type_code IN ('POFO', 'ROSO', 'PFRS', 'INC')
```

Comment:
Is this correct to ignore changes to exclusion above?

Reply:
yes, ticket only specifies these 4 types